staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/grpc_service.go

```diff
@@ -84,15 +86,15 @@ func NewGRPCService(endpoint string, callTimeout time.Duration) (Service, error)
 	}, nil
 }
 
-// Parse the endpoint to extract schema, host or path.
+// Parse the testSocket to extract schema, host or path.
```

Comment:
sorry, a few other comment/message find/replace instances to clean up

Reply:
No, I am sorry. I should have checked the rest of the file.
Fixed.